.vscode/launch.template.json

```diff
@@ -28,6 +28,19 @@
                 "-test.run",
                 "TestSubmodule/${fileBasename}"
             ]
+        },
+        {
+            "name": "Launch submodule test (with profiling)",
+            "type": "go",
+            "request": "launch",
+            "mode": "test",
+            "program": "${workspaceFolder}/internal/testrunner",
+            "args": [
+                "-test.cpuprofile",
+                "test.prof",
```

Comment:
Do you care where this is output? Maybe this should go to `${workspaceFolder}/test.prof`?

Reply:
Done.